README.md

```diff
@@ -2,9 +2,72 @@
 
 An operator to deploy the [OpenStack Cinder CSI driver](https://github.com/openshift/cloud-provider-openstack/tree/master/pkg/csi/cinder) in OpenShift.
 
-# Quick start
+## Configuration
 
-Before running the operator manually, you must remove the operator installed by CSO/CVO
+The operator can be configured using a config map, which by default is found at either `openshift-config / cinder-csi-config` or `openshift-config / cloud-provider-config`.
+The former is preferred as it stores configuration solely for the Cinder CSI Driver.
+It is supported starting in OpenShift 4.14 and should be used in all new deployments.
+As the name might suggest, the latter stores configuration for both the Cinder CSI driver and the OpenStack Cloud Provider.
+It is used for legacy reasons.
+
+Configuration must be stored in the config map under the `config` key. For example:
+
+```shell
+oc get configmap -n openshift-config cinder-csi-config -o yaml
+```
+
+```yaml
+apiVersion: v1
+data:
+  ca-bundle.pem: |
+    <redacted>
+  config: |
+    [Global]
+    ...
+    [BlockStorage]
+    ...
+kind: ConfigMap
+metadata:
+  name: cinder-csi-config
+  namepsace: openshift-config
+```
+
+> *Note*
+> This config map stores configuration for both services for historical reasons: previously, block device management was handled by the cloud provider.
+> This was decoupled in the 4.14 release, but support for the legacy config map is retained to avoid breaking exist deployments.
```

Comment:
> support for the legacy config map is retained

Reading what's above, I thought that there were as of 4.14 two config maps:
1. `cloud-provider-config` contains config for everything, like it always was
2. `cinder-csi-config` contains a subset of what's contained in `cloud-provider-config`, because we decided to decouple that part.

I am confused by your calling the `cloud-provider-config` cm "legacy". Based on what I read, I thought that the `cloud-provider-config` cm contains config that the `cinder-csi-config` cm does not; as a consequence, only some fields of the `cloud-provider-config` cm would be deprecated ("legacy"), but not the whole cm. Did I get this right? If so, to avoid confusion please do not refer to the whole `cloud-provider-config` cm as legacy.